app/uk/gov/hmrc/uknwauthcheckerapi/config/AppConfig.scala

```diff
@@ -19,12 +19,20 @@ package uk.gov.hmrc.uknwauthcheckerapi.config
 import javax.inject.{Inject, Singleton}
 
 import play.api.Configuration
+import uk.gov.hmrc.http.StringContextOps
 
 @Singleton
 class AppConfig @Inject() (config: Configuration) {
 
-  val authType: String = config.get[String]("authType")
+  private lazy val rootServices = "microservice.services"
+  private lazy val defaultProtocol: String =
+    config
+      .getOptional[String](s"$rootServices.protocol")
+      .getOrElse("http")
 
+  val authorisationsEndpoint = "/authorisations"
+  val authType: String = config.get[String]("authType")
+  val eisAuthorisationsUrl = url"${baseUrl("integration-framework")}/cau/validatecustomsauth/v1"
```

Comment:
Dan, can we move /authorisations and /cau/validatecustomsauth/v1 to Application.conf as they might change in future. thanks

Reply:
The endpoint is fixed. It will never change, as it would break existing APIs. Same way we can't change our API context once it's set.